tools/template_based_query_generation/src/main/java/QueryGenerator.java

```diff
@@ -1,31 +1,146 @@
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
+import graph.MarkovChain;
+import graph.Node;
+import parser.*;
+import token.Tokenizer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
 
 /**
  * Class that parses config file and creates queries from markov chain
  */
 public class QueryGenerator {
 
+	private final String filePathConfigDDL = "./src/main/resources/user_config/ddl.json";
```

Comment:
Please fix the indentation of this file to use 2 space of tab

Reply:
Resolved!